api/routes/notification.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import { validateToken } from "../middleware/auth";
+import NotificationService from "../controller/notification"; //  the NotificationService class
+
+const NotificationRouter = express.Router();
+const notificationService = new NotificationService();
+
+// Route handler
+NotificationRouter.post("/email", async (req, res) => {
+  const { email, subject, message } = req.body;
```

Comment:
Always initialize with default value while destructuring.
```suggestion
  const { body = {} } = req; 
  const { email= '', subject = '', message = '' } = body;
```

Reply:
Also please add validation on these field.